scripts/stylelint.js

```diff
@@ -18,4 +18,4 @@
  */
 
 require('../src/setup_node_env');
-require('../src/dev/run_sasslint');
+require('../src/dev/run_stylelint');
```

Comment:
Running this script directly outputs the stylelint help.

Reply:
Pushed d3cedd7.  It uses the stylelint CLI parser to check for input globs, and if there aren't any we default to *.s(c|a)ss.